common/wivrn_packets.h

```diff
@@ -185,6 +186,14 @@ struct hand_tracking
 	std::optional<std::array<pose, XR_HAND_JOINT_COUNT_EXT>> joints;
 };
 
+struct fb_face2
+{
+	std::array<float, 70> weights;
```

Comment:
We could use `XR_FACE_EXPRESSION2_COUNT_FB` if we make sure we have a recent enough OpenXR